etcdctlv3/command/make_mirror_command.go

```diff
@@ -56,11 +56,15 @@ func makeMirrorCommandFunc(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
 		ExitWithError(ExitBadArgs, errors.New("make-mirror takes one destination arguement."))
 	}
+	dialTimeout, err := cmd.Flags().GetDuration("dial-timeout")
```

Comment:
replace this (and the other one) with a call to `dialTimeoutFromCmd(cmd)` in `global.go`?